lib/Form/FormsRepository.php

```diff
@@ -54,4 +54,49 @@ private function updateDeletedAt(FormEntity $form, ?Carbon $value) {
     $this->persist($form);
     $this->flush();
   }
+
+  public function bulkTrash(array $ids): int {
```

Comment:
I think it would be nice to reuse bulkTrash inside public function trash() so that we don't have 2 different implementations for single and bulk action. This applies also to bulkRestore and bulkDelete. That way we could get rid of `updateDeletedAt` method.